src/jobs/run.yml

```
@@ -15,11 +16,11 @@ parameters:
       but before verifying Cypress and saving cache.
     type: string
     default: ""
-  cache-key:
+  cypress-cache-key:
     description: Cache key used to cache the Cypress binary.
     type: string
     default: 'cypress-cache-{{ arch }}-{{ checksum "package.json" }}'
-  cache-path:
+  cypress-cache-path:
     description: |
       By default, this will cache the '~/.cache' directory so that the Cypress binary is cached. You can override this by providing your own cache path.
```

We can update this bit.

```suggestion
      By default, this will cache the '~/.cache/Cypress' directory so that the Cypress binary is cached. You can override this by providing your own cache path.
```